crypto/cryptovm.go

```diff
@@ -0,0 +1,23 @@
+package crypto
+
+import (
+	"github.com/ElrondNetwork/arwen-wasm-vm/crypto/hashing"
+	"github.com/ElrondNetwork/arwen-wasm-vm/crypto/signing/bls"
+	"github.com/ElrondNetwork/arwen-wasm-vm/crypto/signing/ed25519"
+	"github.com/ElrondNetwork/arwen-wasm-vm/crypto/signing/secp256k1"
+)
+
+// NewCryptoVm returns a composite struct containing VMCrypto functionality implementations
+func NewCryptoVm() VMCrypto {
+	return struct {
+		Hasher
```

Comment:
looks strange. maybe Hasher: hashing.NewHasher(),

Reply:
I've added `Hasher:` in the member initialisation. If we add it into the declaration, the struct does not respect the interface anymore. I liked by returning a composite anonymous struct instead of declaring another type just for that. If you still think it's strange and you don't like it, I will make an actual struct for it :)

Regarding the `go mod tidy` comment, I did that, but I think because of the included libraries we have this increase in indirect dependencies. If you run `go mod tidy` you get a smaller result for `go.sum`?